plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-order-summary-subtotal/block.tsx

```diff
@@ -5,13 +5,26 @@ import { Subtotal, TotalsWrapper } from '@woocommerce/blocks-components';
 import { getCurrencyFromPriceResponse } from '@woocommerce/price-format';
 import { useStoreCart } from '@woocommerce/base-context/hooks';
 
-const Block = ( { className = '' }: { className?: string } ): JSX.Element => {
+export type BlockAttributes = {
+	className?: string;
+	heading?: string;
```

Comment:
```suggestion
	heading: string;
```

Shouldn't the heading attribute be required? It's either the custom heading of the merchant or the placeholder, isn't it?

Reply:
Dang thanks for spotting this, looks like I relied too much on the AI to generate these type defs, whoops. Will fix.

Reply:
As for my understanding, should it only be `string | null` when we allow displaying an empty string on the frontend? This goes back to my question in https://github.com/woocommerce/woocommerce/pull/53544#discussion_r1875943473. Or should this definition be  `string | null` independent of us allowing merchants to display empty strings?

Reply:
@nielslange in light of changing the UX experience we will change this for sure. but if the JSON definition had a default of `null` then this would be `string | null`. Maybe that's obvious but just pointing that out. The type definition will match how the attribute is defined.